cognite/neat/core/rules/exporter/rules2graphql.py

```diff
@@ -1,21 +1,99 @@
-import logging
-import re
-import warnings
-
-from graphql import GraphQLField, GraphQLList, GraphQLNonNull, GraphQLObjectType, GraphQLSchema, print_schema
-
 from cognite.neat.core.rules import _exceptions
-from cognite.neat.core.rules.analysis import get_classes_with_properties
+from jinja2 import DictLoader, Environment
 from cognite.neat.core.rules._validation import (
     are_entity_names_dms_compliant,
     are_properties_redefined,
 )
-from cognite.neat.core.rules.models import DATA_TYPE_MAPPING, Property, TransformationRules
+
+from cognite.neat.core.rules.analysis import get_class_property_pairs
+from cognite.neat.core.rules.models import DATA_TYPE_MAPPING, TransformationRules
 from cognite.neat.core.utils.utils import generate_exception_report
 
 
+TYPE = (
```

Comment:
I suggest making all these constants private, as they should not be used outside of this module. 
```suggestion
_TYPE = (
```

Reply:
Agree, implemented